mw-webapp/src/logic/waysTable/MentoringWaysTable.module.scss

```diff
@@ -4,3 +4,9 @@
   margin-left: 5px;
   cursor: pointer;
 }
+
+.modalButtons {
```

Comment:
Why do you need this stones, if you don't have appropriate modal component?

Reply:
Removed